docs/resources/project.md

```diff
@@ -4,14 +4,45 @@ page_title: "project Resource - terraform-provider-project"
 subcategory: ""
 description: |-
   Provides an Artifactory project resource. This can be used to create and manage Artifactory project, maintain users/groups/roles/repos.
-  ~>We strongly recommend using the "repos" attribute to manage the list of repositories. See below for more details.
+  Repository Configuration
+  After the project configuration is applied, the repository's attributes project_key and project_environments would be updated with the project's data. This will generate a state drift in the next Terraform plan/apply for the repository resource. To avoid this, apply lifecycle.ignore_changes:
```

Comment:
it's odd that doc plugin removed quotes here: `attributes project_key and project_environments would`

Reply:
Yeah, since these are in the YAML Frontmatter section they don't affect the actual documentation.